version.inc

```diff
@@ -9,7 +9,7 @@ else
     logprint "Current MailScanner version is $MailScannerVersion"
 fi
 
-MailWatchVersion=$(wget https://github.com/mailwatch/mailwatch/releases -q -O - | grep '<span class="css-truncate-target">' | grep -vi 'rc' | head -n1 | cut -d'>' -f2 | cut -d'<' -f1)
+MailWatchVersion=$(wget https://github.com/mailwatch/mailwatch/releases -q -O - | grep '<span class="css-truncate-target"' | grep -vi 'rc' | head -n1 | cut -d'>' -f2 | cut -d'<' -f1)
```

Comment:
what about this?
`curl --silent "https://api.github.com/repos/mailwatch/mailwatch/releases/latest" | grep  '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'`